Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14
+
+LABEL maintainer="Ryan Blunden <ryan.blunden@ext.doppler.com>"
+
+# DOPPLER_TOKEN must be supplied as a build-arg
+ARG DOPPLER_TOKEN
+ENV DOPPLER_TOKEN ${DOPPLER_TOKEN}
```

Comment:
I'm not familiar with `ENV` immediately following `ARG`. Is this so that the build arg is also available at runtime?

Reply:
Yep, that's the one.

Reply:
I've removed this now as if we're pushing this to our DockerHub account, then the `DOPPLER_TOKEN` needs to be supplied at runtime, not build-time.

Reply:
I wasn't familiar with this paradigm- thanks for the knowledge drop!